back/src/commands/onboarding.helpers.ts

```diff
@@ -112,3 +113,47 @@ export const sendSecondOnboardingEmail = async () => {
   );
   await prisma.$disconnect();
 };
+
+// Retrieve users:
+// - who are active
+// - whose account was created x daysAgo
+// - who never issued a membership request
+// - who do not belong to a company
+export const getRecentlyRegisteredUsersWithNoCompanyNorMembershipRequest =
+  async (daysAgo: number) => {
+    const now = new Date();
+
+    const associatedDateGt = xDaysAgo(now, daysAgo);
+    const associatedDateLt = xDaysAgo(now, daysAgo - 1);
+
+    const users = await prisma.user.findMany({
+      where: {
+        createdAt: { gt: associatedDateGt, lt: associatedDateLt },
+        isActive: true,
+        companyAssociations: { none: {} },
+        MembershipRequest: { none: {} }
+      }
+    });
+
+    return users;
+  };
+
+/**
+ * Send a mail to users who registered recently and who haven't
+ * issued a single MembershipRequest yet
+ */
+export const sendMembershipRequestDetailsEmail = async () => {
+  const recipients =
+    await getRecentlyRegisteredUsersWithNoCompanyNorMembershipRequest(7);
+
+  await Promise.all(
+    recipients.map(recipient => {
+      const payload = renderMail(membershipRequestDetailsEmail, {
+        to: [{ email: recipient.email, name: recipient.name }]
+      });
+
+      return sendMail(payload);
+    })
+  );
```

Comment:
np, mais on peut pas faire:
```javascript
const payload = renderMail(membershipRequestDetailsEmail, {
        to: [recipients.map(recipient => ({ email: recipient.email, name: recipient.name })}]
 })
await sendMail(payload);
```
P-e pas avec un `to` mais plutot un `cc`. Enfin il ne fauit pas partager les adresses mails à tout le monde, je ne sais pas comment l'api SIB le gère

Reply:
C'est une excellente remarque! Je ne vois aucun endroit dans le code où on fait des envois groupés, est-ce qu'on a un cador SIB dans la team pour en discuter ?

Reply:
En effet on envoi rarement des mails en groupe, mais pour ce genre de use cases ca serait cool. Je ne suis pas expert SIB, l'API semble bien supporter l'envoi groupé mais comme je disais il faut juste être vigilant à ne pas partager tous les mails dans le `to`.
Peut être que le param `messageVersion` serait à utiliser ici ? cf https://developers.sendinblue.com/reference/sendtransacemail
A tester je dirais, ou alors un autre dev sait p-e

Reply:
Sendinblue permet effectivement de batcher https://developers.sendinblue.com/docs/batch-send-transactional-emails avec messageVersion, ça me semble préférable à un cc. Ça va nécessiter qq adaptations sur le moteur console